app/screens/Settings/SettingsScreen.tsx

```diff
@@ -91,6 +92,8 @@ const SettingsScreen = ({ navigation }: Props) => {
           </Button.Primary>
         </View>
       ) : null}
+
+      <Text.Tertiary style={styles.appVersionTitle}>{t("SETTINGS_SCREEN_APP_VERSION", { version: AppInfos.expo.version })}</Text.Tertiary>
```

Comment:
You can pass bold in any Text element by passing it as a prop e.g : <Text.Tertiary bold style={styles.appVersionTitle}>